TACC/Core-Styles
# fix: disabled button text too hard to see

## Overview

Make text within busy button easier to see.

## Related

- benefits #__

## Changes

- **changed** `opacity` of text within button

## Testing

1. Open http://localhost:3000/components/detail/c-button--primary.
2. Verify text in `--is-busy` button is easier to see than before.

## UI

| before | after |
| - | - |
| <img width="1194" alt="before" src="https://github.com/user-attachments/assets/c82746f5-7553-4d94-9a07-84c7c84e0e84"> | <img width="1194" alt="after" src="https://github.com/user-attachments/assets/c3572ecc-f686-4d72-be99-cc072fb240d8"> |
